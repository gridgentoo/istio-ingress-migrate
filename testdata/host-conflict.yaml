apiVersion: v1
items:
- apiVersion: extensions/v1beta1
  kind: Ingress
  metadata:
    annotations:
      kubernetes.io/ingress.class: istio
    name: http-only
    namespace: default
  spec:
    rules:
    - host: service.com
      http:
        paths:
        - backend:
            serviceName: echo
            servicePort: 80
          path: /*
          pathType: ImplementationSpecific
- apiVersion: networking.istio.io/v1alpha3
  kind: Gateway
  metadata:
    name: istio-autogenerated-k8s-ingress
    namespace: gke-system
  spec:
    selector:
      istio: ingress-gke-system
    servers:
    - hosts:
      - 'service.com'
      port:
        name: http
        number: 443
        protocol: HTTP2
      tls:
        mode: SIMPLE
        credentialName: secret
    - hosts:
      - 'service.com'
      port:
        name: http
        number: 443
        protocol: HTTP2
      tls:
        mode: SIMPLE
        credentialName: secret
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""

