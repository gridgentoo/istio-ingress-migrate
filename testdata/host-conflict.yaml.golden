ERR04: Gateway istio-autogenerated-k8s-ingress server for host service.com conflicts with another gateway
---
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: istio
  creationTimestamp: null
  name: http-only
  namespace: default
spec:
  rules:
  - host: service.com
    http:
      paths:
      - backend:
          serviceName: echo
          servicePort: 80
        path: /*
        pathType: ImplementationSpecific
  tls:
  - hosts:
    - service.com
    secretName: secret
status:
  loadBalancer: {}

failures detected during execution
